draft-ietf-httpbis-http2-encryption.md

```diff
@@ -210,39 +215,51 @@ representing an interval in seconds.
 }
 ~~~
 
+Including `commit` creates a commitment to provide a secured alternative service for the advertised
+period. Clients that receive this commitment can assume that a secured alternative service will be
+available for the indicated period. Clients might however choose to limit this time (see
+{{pinrisks}}).
+
+## Client Handling of A Commitment
 
 The value of the `commit` member MUST be ignored unless the alternative service can be strongly
-authenticated.  Minimum authentication requirements for HTTP over TLS are described in Section 2.1
-of {{I-D.ietf-httpbis-alt-svc}} and Section 3.1 of {{RFC2818}}.  As noted in
-{{I-D.ietf-httpbis-alt-svc}}, clients can impose other checks in addition to this minimum set.  For
+authenticated. The same requirements that apply to `https://` resources SHOULD be applied to
```

Comment:
I'm concerned that this is going to confuse people -- *which* same requirements?